### ISSUE TITLE
Add command to retrieve all or some info from JIRA issue (e.g. custom fields)

### ISSUE THREAD

**Issue by ggruen** [cloudily/jira#3]
If we could retrieve information about a specified JIRA issue, especially custom fields (e.g. "Budget Code" or "Requestor"), we could use that information for reporting or for tying into other systems.

**Comment by cloudily** [comment]
Makes sense.  Right now, I only exposed `issue_worklog`, but other resources can be mapped.  Here is the endpoint that you would use to get issue data: https://docs.atlassian.com/jira/REST/latest/#api/2/issue-getIssue

If you have some time and would like to map `issue` (or even just `jira.issue.get`), I would be happy to merge it.
